src/App.js

```diff
@@ -0,0 +1,74 @@
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
+import NavBar from './components/NavBar';
+import testImg from './assets/images/test-img.png';
+import Photos from './pages/Photos';
+import axios from 'axios';
+import { useEffect, useState } from 'react';
+import NoResult from './components/NoResult';
+import Loader from './components/Loader';
+
+const API_URL = 'https://jsonplaceholder.typicode.com/albums/1/photos';
+
+const App = () => {
+  const [photos, setPhotos] = useState([]);
+  const [filteredValue, setFilteredValue] = useState('');
+  const [isError, setIsError] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const checkErrorsFromAPI = (response) => {
```

Comment:
This function will be used also for other requests so let's move it to separate file